trview.app/Geometry/PickResult.cpp

```diff
@@ -147,6 +148,14 @@ namespace trview
                 }
                 break;
             }
+            case PickResult::Type::FlybyNode:
+            {
+                if (const auto flyby_node = result.flyby_node.lock())
+                {
+                    stream << "Flyby Camera " << flyby_node->number();
```

Comment:
Would it be feasible to show the main flyby index as well as the node? E.g. `Flyby Camera 1, Node 3`